sdk/keyvault/azure.keyvault.keys.txt

```diff
@@ -0,0 +1,180 @@
+**** Package/Module azure.keyvault.keys****
+class KeyClient(KeyVaultClientBase, object):
+	vault_url:str
+	credential
+
+	vault_url
+
+	def __init__(self, vault_url: str, credential, transport=None, api_version=None, **kwargs): ...
+	def backup_key(self, name: str, **kwargs) -> bytes: ...
```

Comment:
"backup" is a verb here? Maybe something like `create_key_backup` (or even just `create_backup`?)

Reply:
Same comment as on SecretClient unless we have a strong reason.